Dockerfile

```diff
@@ -47,7 +45,7 @@ RUN mkdir -p /etc/laminas-ci/problem-matcher \
 
 COPY etc/markdownlint.json /etc/laminas-ci/markdownlint.json
 
-COPY --from=composer /usr/bin/composer /usr/bin/composer
+COPY --from=composer:2.2.4 /usr/bin/composer /usr/bin/composer
```

Comment:
Overall OK with this, but we need to add `dependabot.yml` for bumping this regularly